tests/pester/release.tests.ps1

```diff
@@ -0,0 +1,58 @@
+[CmdletBinding()]
+param (
+  [Parameter(Mandatory = $false)]
+  [string]
+  $versionFilePath = "./version.json"
```

Comment:
Consider combining version and gitTag in this file. They are the same save for a v prefix. This is simple to do in code and removes the need to update two values that could get out of sync.

Reply:
will remove gitTag